packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-3">Import</button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-0-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-0-0">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-0-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-1" hidden="">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-0-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-0-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</figure>
+
+```html
+    <div class="ods-tabs" id="example-0">
+      <div class="ods-tablist" role="tablist" aria-label="">
+        <button class="ods-tab" id="tablistitem-0-0" role="tab" aria-selected="false" aria-controls="tab-0-0">
+          Agents
+        </button>
+        <button class="ods-tab" id="tablistitem-0-1" role="tab" aria-selected="false" aria-controls="tab-0-1">
+          People
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-2">
+          Settings
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-3">
+          Import
+        </button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-0-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-0-0">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-0-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-1" hidden="">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-0-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-0-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+```
+## Anatomy
+
+### Tablist
+The tablist is the parent element that houses tabs. It provides no style in and of itself.
+
+### Tab
+The tab is the element in which the user clicks to change to the corresponding tabpanel.
+
+### Tab indicator
+The tab indicator is a pseudo-element used to indicate the active tab. Its position changes by calculating the left offset, as well as the width of the active tab element. (See [Switching Tabs](#switching-tabs))
+
+### Tabpanel
+A tabpanel is a simple container element. For each tab, there is an associated tab panel which contains the content associated with that tab.
+
+## Switching tabs
+
+When a tab is selected, three things must happen:
+
+1. The indicator must change position and width.
```

Comment:
Since the audience here seems to be Eng, maybe something more direct, "Update the CSS custom properties to animate the Tab indicator correctly. (See below)"?